workers/unity/Assets/Playground/Scripts/Player/ProcessLaunchCommandSystem.cs

```diff
@@ -18,7 +18,8 @@ private struct LaunchCommandData
             [ReadOnly] public EntityArray Entity;
             public ComponentDataArray<SpatialOSLauncher> Launcher;
 
-            [ReadOnly] public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
+            [ReadOnly]
+            public ComponentArray<CommandRequests<Generated.Playground.Launcher.LaunchEntity.Request>> CommandRequests;
```

Comment:
Is this the linter flip-flopping again

Reply:
looks like it 🤔

Reply:
I hit this on my linting as well. It looks like there is some discrepancy about line length somewhere

Reply:
It looks like there not a setting in the `DotSettings` for the line length

Reply:
Just tested, and adding it to the `.editorconfig` works. Will open a separate PR for that

Reply:
Sweet :D